index.html

```diff
@@ -306,6 +306,14 @@ <h2>
             being used in a 'laptop' posture.
             </li>
           </ul>
+          <div class="note">
+            <p>Even if a foldable is used in a half-folded state like a book, text direction needs
```

Comment:
"Text direction" here is a bit confusing. Generally this term refers to the differences in left-to-right vs. right-to-left writing systems (such as the use of the Arabic script for many languages). It is the case that page-progression is linked to text direction, although note that the CJK languages are all typically left-to-right. The vertical writing mode used by these languages is what progresses differently.

Minor English nit: _was being used_ should be _were being used_

What I'd suggest is maybe mentioning this more directly. Perhaps:

```suggestion
          <p>When a foldable is used in a half-folded state (like a book), 
             implementers might need to take into account the effect of
             text direction and writing mode on the layout and presentation.
          <p>For example, right-to-left languages 
             (those that use scripts such as Arabic, Hebrew, Syriac, and others) 
             and many vertical writing modes used by, for example, East Asian languages
             progress pages in the opposite order to that used by an English book, 
             with the lower numbered page on the right.
             See: [links to CJK]
```